bin/link-changelog

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# link creates a link to changelog for the package's version on https://pub.dev.
+#
+# It assumes that the git tag starts with "v".
+
+package_name="${1:-}"
+tag="${2:-}"
+prefix="${3:-}"
+
+if [ -z "$package_name" ]; then
+    echo "error: missing package name"
+    exit 1
+
+fi
+
+if [ -z "$tag" ]; then
+    echo "error: missing tag"
+    exit 1
+fi
+
+if [ -n "$prefix" ]; then
+    tag="${tag//$prefix/}"
+fi
+
+version=$(echo "$tag" | cut -c 2- | sed 's/\.//g' | sed 's/\+//g')
```

Comment:
Two things:

* `cut` will always cut the first character, no matter what it is. How about cutting just `v` instead?
* `sed` appears to be abused in order to just remove all instances of two characters from the result. How about using `tr -d` instead?

```suggestion
version=$(echo "${tag#v}" | tr -d '.+')
```

Reply:
Thanks c: